server/src/routes/api/v1/squidsRouter.js

```diff
@@ -8,7 +8,14 @@ export const squidsRouter = new express.Router();
 squidsRouter.get(
   "/",
   nextWrapper(async (req, res) => {
-    const squids = await Squid.query().orderBy("createdAt", "desc");
-    res.status(200).json({ squids });
+    const pageNumber = req.query.pageNumber - 1;
+    const resultsPerPage = 10;
+
+    const squidsQueryData = await Squid.query()
+      .orderBy("id", "asc")
+      .page(pageNumber, resultsPerPage);
+    const pages = Math.ceil(squidsQueryData.total / resultsPerPage);
+
+    res.status(200).json({ squidsQueryResults: { squidsData: squidsQueryData.results, pages } });
```

Comment:
@CassandraNewell would love to chat further on this weird issue with sorting by createdAt. For now it works best with id

Reply:
lemme know if you have any follow-up questions after our brief Slack chat!